src/status_im2/contexts/add_new_contact/events.cljs

```diff
@@ -6,114 +6,133 @@
             [status-im.ethereum.ens :as ens]
             [status-im.ethereum.stateofus :as stateofus]
             [status-im.native-module.core :as status]
-            [status-im2.navigation.events :as navigation]
-            [status-im2.utils.validators :as validators]
+            [status-im2.utils.validators :as v]
```

Comment:
we're agreed to use full names `as validators`

Reply:
agreed. will fix.

Reply:
done